test/utils/LoggingInterface.spec.ts

```diff
@@ -94,10 +94,10 @@ describe('LoggingInterface', function() {
       warnMock.mockClear();
       errorMock.mockClear();
     });
-    test('debug', function() {
-      Logger.debug("test", {}, "test2");
+    test('debug', () => {
+      Logger.debug('test', {}, 'test2');
       expect(debugMock).toHaveBeenCalledTimes(1);
-      expect(debugMock).toHaveBeenCalledWith("test", {}, "test2");
+      expect(debugMock).toHaveBeenCalledWith('test', {}, 'test2');
       expect(infoMock).toHaveBeenCalledTimes(0);
```

Comment:
This is overkill :D

Reply:
It is a simple `--fix` and I gotta admit I like that its constant 😄